src/main/rules/GCI95/python/GCI95.asciidoc

```diff
@@ -0,0 +1,63 @@
+CSV for Comma-Separated Values is a widely-used way of storing data. Its main advantage is that it is “human readable”, meaning that a human can understand the data. However, the fact that it is stored per line and in “plain text” can lead to performance problems.
```

Comment:
I think, this is a real rule for IA or Data subject, but not OK for standard use cases for example.
Because this subject can be extended to other plain text formats like JSON, or simple property files that are used for an application configuration for example. And in these use-cases, compression file is not recommanded.

Reply:
CSV is not a best partice for configuration (prefer YAML or JSON). the use case here, is to replace CSV to Parquet (because CSV is commonly used to store data and not configuration)